Makefile

```diff
@@ -19,13 +19,7 @@ define build-image
 	docker system prune --all -f
 endef
 
-all: 10 11 12 13 14
-
-10:
```

Comment:
Build for version 10 and 11 does not work anymore ? Can you provide details on the 10 and 11 removal ?

Reply:
Version 10 was not building because `postgres:10-bookworm` does not exist. Because of that I have removed 10 and 11 as they are not supported anymore by PostgreSQL

Reply:
I've put back version 10 and 11, using bullseye for version 10 and bookworm otherwise

Reply:
Ok Let's do it this way